src/utils/translate.js

```diff
@@ -0,0 +1,11 @@
+// @flow
+import config from '../../config.json';
+import {translate} from '../i18n';
+import type {Language} from '../i18n';
+
+export default function t(
+    key: $Keys<Language>,
+    substitutions?: {[string]: string},
+): string {
+    return translate(config.language)(key, substitutions);
```

Comment:
Based on previous comment, I think is better check language before starting.

Maybe a `localeExists(language)` that throws an error. If app starts, we're safe with localized strings